### PR TITLE
_reports/RJ722.md: Update report for RJ722

### PR DESCRIPTION
**Pull request by RJ722**
* Add permalink
* Add new commit

### CODE REVIEW

File: _reports/RJ722.md [hunk 1]
```diff
@@ -266,61 +267,66 @@ activity:
       - details: "Add --ignore-names flag (#139)."
 
   - 46:
+      - repo: vulture
+      - link: https://github.com/jendrikseipp/vulture/commit/3c965e2
+      - details: "Add Item.get_whitelist_string. (#144)"
```
Comment (jayvdb) [inline _reports/RJ722.md]: There are three commits in this PR, and they also need to be added to your tarball if you want to add them to the activity report.

Comment (RJ722) [reply _reports/RJ722.md]: I've added the recent ones from Vulture. Anything else left?

Comment (jayvdb) [reply _reports/RJ722.md]: Ah I see , they are squashing into one commit.

Ok, then this checks out.  Thanks

**Comment by jayvdb** [comment]
ack https://github.com/coala/projects/pull/639/commits/af9193497fd905b870c33ab4b08d8b9cd8d4c9a7

**Comment by jayvdb** [comment]
@gitmate-bot ff

**Comment by jayvdb** [comment]
@gitmate-bot ff

**Pull request merged**